### PR TITLE
Updating java.nio.file

### PR DESCRIPTION
**Pull request by albertomisail**

### CODE REVIEW

File: jre_emul/tests.mk [hunk 1]
```diff
@@ -338,7 +338,8 @@ $(TESTS_DIR)/core_plus_concurrent:
 
 $(TESTS_DIR)/core_plus_file:
 	@mkdir -p $(@D)
-	$(J2OBJCC) -ljre_file -o $@ -ObjC
+	$(J2OBJCC) -ljre_file -ljre_concurrent -ljre_net -ljre_security \
+		-ljre_util -o $@ -ObjC
```
Comment (tomball) [inline jre_emul/tests.mk:342]: Nit: indent -4.

File: jre_emul/tests.mk [hunk 2]
```diff
@@ -355,7 +356,7 @@ $(TESTS_DIR)/core_plus_ssl:
 $(TESTS_DIR)/core_plus_time:
 	@mkdir -p $(@D)
 	$(J2OBJCC) -ljre_time -ljre_icu -ljre_channels -ljre_net -ljre_util \
-	    -ljre_security -ljre_zip -ljre_io -ljre_concurrent -o $@ -ObjC
+		-ljre_security -ljre_zip -ljre_io -ljre_concurrent -o $@ -ObjC
```
Comment (tomball) [inline jre_emul/tests.mk:359]: Nit: indent -4 (undo edit).

File: jre_emul/tests.mk (hunk 1 above)
Comment (albertomisail) [reply jre_emul/tests.mk]: Fixed

File: jre_emul/tests.mk (hunk 2 above)
Comment (albertomisail) [reply jre_emul/tests.mk]: Fixed

**Pull request merged**